frontend/src/container/PlannedDowntime/PlannedDowntime.tsx

```diff
@@ -0,0 +1,211 @@
+import './PlannedDowntime.styles.scss';
+import 'dayjs/locale/en';
+
+import { PlusOutlined } from '@ant-design/icons';
+import { Color } from '@signozhq/design-tokens';
+import {
+	Button,
+	Dropdown,
+	Flex,
+	Form,
+	Input,
+	MenuProps,
+	Typography,
+} from 'antd';
+import getAll from 'api/alerts/getAll';
+import { useDeleteDowntimeSchedule } from 'api/plannedDowntime/deleteDowntimeSchedule';
+import {
+	DowntimeSchedules,
+	GetAllDowntimeSchedulesPayloadProps,
+	useGetAllDowntimeSchedules,
+} from 'api/plannedDowntime/getAllDowntimeSchedules';
+import dayjs from 'dayjs';
+import { useNotifications } from 'hooks/useNotifications';
+import { CalendarClockIcon, PencilRuler, Search, SortDesc } from 'lucide-react';
+import React, { ChangeEvent, useState } from 'react';
+import { useQuery } from 'react-query';
+
+import { PlannedDowntimeDeleteModal } from './PlannedDowntimeDeleteModal';
+import { PlannedDowntimeForm } from './PlannedDowntimeForm';
+import { PlannedDowntimeList } from './PlannedDowntimeList';
+import {
+	defautlInitialValues,
+	deleteDowntimeHandler,
+} from './PlannedDowntimeutils';
+
+export function PlannedDowntime(): JSX.Element {
+	const { data, isError, isLoading } = useQuery('allAlerts', {
+		queryFn: getAll,
+		cacheTime: 0,
+	});
+	const [isOpen, setIsOpen] = React.useState(false);
+	const [form] = Form.useForm();
+
+	const [initialValues, setInitialValues] = useState<
+		Partial<DowntimeSchedules & { editMode: boolean }>
+	>(defautlInitialValues);
+
+	const [
+		downtimeSchedulePayload,
+		setDowntimeSchedulePayload,
+	] = useState<GetAllDowntimeSchedulesPayloadProps>({
+		acitve: false,
+		recurrence: false,
+	});
+	const downtimeSchedules = useGetAllDowntimeSchedules(downtimeSchedulePayload);
```

Comment:
do we not want error state from this API ?

Reply:
Added error notification in listing page